src/libstd/sys/windows/ext/process.rs

```diff
@@ -97,3 +97,32 @@ impl ExitStatusExt for process::ExitStatus {
         process::ExitStatus::from_inner(From::from(raw))
     }
 }
+
+/// Windows-specific extensions to the `std::process::Command` builder
+#[unstable(feature = "windows_process_extensions", issue = "37827")]
+pub trait CommandExt {
+    /// Sets the [process creation flags][1] to be passed to `CreateProcess`.
+    ///
+    /// These will always be ORed with `CREATE_UNICODE_ENVIRONMENT`.
+    /// [1]: https://msdn.microsoft.com/en-us/library/windows/desktop/ms684863(v=vs.85).aspx
+    #[unstable(feature = "windows_process_extensions", issue = "37827")]
+    fn set_creation_flags(&mut self, flags: u32) -> &mut process::Command;
+    /// Add `flags` to the the [process creation flags][1] to be passed to `CreateProcess`.
+    ///
+    /// These will always be ORed with `CREATE_UNICODE_ENVIRONMENT`.
+    /// [1]: https://msdn.microsoft.com/en-us/library/windows/desktop/ms684863(v=vs.85).aspx
+    #[unstable(feature = "windows_process_extensions", issue = "37827")]
+    fn add_creation_flags(&mut self, flags: u32) -> &mut process::Command;
```

Comment:
I renamed it to `creation_flags` (with an s) because the parameter in the `CreateProcess` signature is named `dwCreationFlags`. (There seem to be a few `_flags` methods in `OpenOptions` as well.)